logic/broadcaster.js

```diff
@@ -240,13 +226,39 @@ Broadcaster.prototype.maxRelays = function(object) {
 	}
 
 	if (Math.abs(object.relays) >= self.config.relayLimit) {
-		library.logger.debug('Broadcast relays exhausted', object);
+		library.logger.info('Broadcast relays exhausted', object);
```

Comment:
This log and below ones which I have changed from `debug` to `info` is because they appear to be more informational statements concerning program state rather than debugging information. `Broadcast relays exhausted` is more of informational statements, whats your take?

Reply:
My understanding is that this log will be displayed for each transaction once relay has exhausted. Which will only be useful in case we are debugging. So, I would keep it at `debug` level.